### ISSUE TITLE
Vue3, Quasar: module not found

### ISSUE THREAD

**Issue by Dimonka2** [bcakmakoglu/vue-flow#29]
Hi, 
I am trying to add vue-flow into a Quasar app and getting a strange error message as soon as I include VueFlow as a component to a vue template script:

> ERROR in ./src/components/New.vue?vue&type=script&lang=ts (./node_modules/@quasar/app/lib/webpack/loader.js.transform-quasar-imports.js!./node_modules/ts-loader/index.js??clonedRuleSet-3.use[0]!./node_modules/@quasar/app/lib/webpack/loader.vue.auto-import-quasar.js??ruleSet[0].use[0]!./node_modules/vue-loader/dist/index.js??ruleSet[0].use[1]!./src/components/New.vue?vue&type=script&lang=ts) 3:0-42
> 
> Module not found: Error: Default condition should be last one
> 

The error looks a bit weird and google unable help me.. Do you have any idea how to solve this error?
The import code is taken from samples. and line `components: {}` breaks compilation:

```js
import { VueFlow, Elements } from '@braks/vue-flow'

export default defineComponent({
  components: { VueFlow },
```

Thanks!

**Comment by bcakmakoglu** [comment]
Ah, I think I know what the issue here is though there's not fix for it at the moment, I am really sorry about that.
*BUT!* It will be fixed in the upcoming v0.4.0 Release which should be done in the next couple weeks - the only thing really missing is the documentation.

Again: Sorry for the inconvenience.

**Comment by Dimonka2** [comment]
> Again: Sorry for the inconvenience.
No problem. 
Just this component is exactly what I was looking for. And there is no real free alternative for it..
Hopefully you will manage to make it compatible with Quasar.

**Comment by bcakmakoglu** [comment]
This got auto-closed with merge - was unintended.
I will leave this open until 0.4.0 is actually released.

@Dimonka2 
I just published the pre-release v0.4.0-1 (i.e. the @next-Tag) which you can use with Quasar. I just created an example and it worked with no issues 🚀
<img width="851" alt="image" src="https://user-images.githubusercontent.com/78412429/155020584-2c6c87f3-bb2b-4873-a0f3-2dda8ec22648.png">

Just add this to your dependencies.
```json
  "dependencies": {
    "@braks/vue-flow": "^0.4.0-1",
    }
```

There is no documentation for all the 0.4.0 features yet *but* basically all the features in 0.3.x work in 0.4.0 as well (some properties might have been renamed, as well as events but you can just check the examples in this repo to find out how it should work).

Make sure to add the necessary styles and provide an actual height to the flow container. Quasar's default dashboard comes with a flex container which will make it look like there is no Flow component. 
```css
@import '~@braks/vue-flow/dist/style.css';
@import '~@braks/vue-flow/dist/theme-default.css';

main {
  height: 100vh;
}
```

**Comment by Dimonka2** [comment]
Hi @bcakmakoglu , 
Now I am getting an error during npm installation of package:
`verbose stack Error: Unsupported URL Type "link:": link:.`

Googling it seems to show me that `yarn` does support "link" as protocol, but `npm` requires "file". Or maybe I am misinterpreting what I have read.

**Comment by bcakmakoglu** [comment]
@Dimonka2 

Try release 0.4.0-2 and let me know if that still throws errors.

**Comment by Dimonka2** [comment]
@bcakmakoglu 
Now it look like compiled and even can be inserted into the app, but in dev mode I got following:

```
 Module not found: Error: Can't resolve '~@braks/vue-flow/dist/style.css'
 Module not found: Error: Can't resolve '~@braks/vue-flow/dist/theme-default.css
```

**Comment by bcakmakoglu** [comment]
Try 
```css
@import 'node_modules/@braks/vue-flow/dist/style.css';
@import 'node_modules/@braks/vue-flow/dist/theme-default.css';
```

**Comment by Dimonka2** [comment]
Currently it got to work like this:

```
@import 'src/../node_modules/@braks/vue-flow/dist/style.css';
@import 'src/../node_modules/@braks/vue-flow/dist/theme-default.css';
```

But I cannot see anything on the screen.. 
Here is a piece of a template where I am inserting VueFlow:

```vue
    <q-tab-panels v-model="activeGroup" animated>
      <q-tab-panel v-for="grp in allGroups" :key="grp.id" :name="grp.id" >
        <div class="text-h6">{{grp.name}}</div>
        <div style="height: 400px;">
          <VueFlow :elements="elements" />
        </div>
      </q-tab-panel>
    </q-tab-panels>
  </div>
</template>

<script lang="ts">
import { PatientGroup } from 'src/model/patient_group'
import { defineComponent, ref } from 'vue'
import { mapState } from 'vuex'
import { VueFlow, Position } from '@braks/vue-flow'


export default defineComponent({
  components: {  VueFlow },
  data() {
    return {
      elements: [
        {
          id: '1',
          data: {
            label: 'node 1',
          },
          position: { x: 100, y: 100 },
          targetPosition: Position.Right
        },
        {
          id: '2',
          data: {
            label: 'node 2',
          },
          position: { x: 100, y: 200 },
          sourcePosition: Position.Left
        },
        {
          id: 'e1-2',
          label: 'default edge',
          target: '2',
          source: '1',
        },
      ]
    }
  },
```

**Comment by bcakmakoglu** [comment]
Elements has been deprecated as a prop, use either v-model or `:nodes="nodes"` and `:edges="edges"`
```vue
 <VueFlow v-model="elements" />
```

**Comment by Dimonka2** [comment]
@bcakmakoglu  Thank you so much! It is working now!

**Comment by bcakmakoglu** [comment]
@Dimonka2 
Glad I could help! :)

**Comment by bcakmakoglu** [comment]
Closed and fixed with #62
